webview-ui/src/components/IncidentTable/IncidentTable.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { FC, useCallback } from "react";
+import { Content, Button, Card, CardBody, CardHeader } from "@patternfly/react-core";
+import { Incident } from "@editor-extensions/shared";
+import { Table, Thead, Tr, Th, Tbody, Td, TableText } from "@patternfly/react-table";
+import * as path from "path-browserify";
+import ViolationActionsDropdown from "../ViolationActionsDropdown";
+
+export interface IncidentTableProps {
+  workspaceRoot: string;
+  incidents: Incident[];
+  message: string;
+  getSolution: (incidents: Incident[]) => void;
+  onIncidentSelect: (it: Incident) => void;
+}
+
+export const IncidentTable: FC<IncidentTableProps> = ({
+  incidents,
+  message,
+  getSolution,
+  workspaceRoot,
+  onIncidentSelect,
+}) => {
+  const fileName = (incident: Incident) => path.basename(incident.uri);
+  const relativeDirname = useCallback(
+    (incident: Incident) => {
+      const dir = path.dirname(incident.uri);
+      const re = new RegExp(`^${workspaceRoot}\\/*`);
+      return dir.replace(re, "");
+    },
+    [workspaceRoot],
+  );
+  const uniqueId = (incident: Incident) => `${incident.uri}-${incident.lineNumber}`;
+
+  const ISSUE = "Issue";
+  const LOCATION = "Location";
+  const FOLDER = "Folder";
+  return (
+    <>
+      <Card isPlain>
+        <CardHeader
+          actions={{
+            actions: (
+              <ViolationActionsDropdown
+                onGetAllSolutions={() => getSolution(incidents)}
+                fixMessage={
+                  incidents.length === 1
+                    ? "Resolve 1 incident"
+                    : `Resolve the ${incidents.length} incidents`
+                }
+              />
+            ),
+          }}
+        />
+
+        <CardBody>{message}</CardBody>
```

Comment:
Could use a label for this message.

Reply:
The label does have the "Show more" but also has a border that we would need to remove manually.  Using plain text seemed the closest to [wireframe 17](https://github.com/konveyor/editor-extensions/blob/main/docs/roadmap/wireframes/17.issues-view-webview.excalidraw.svg)

Reply:
Implemented as suggested in commit https://github.com/konveyor/editor-extensions/pull/195/commits/97b41a69e9e14a1723db6a7dae6d9c1c08efe141

New look:
![Screenshot from 2025-01-15 20-03-07](https://github.com/user-attachments/assets/8e721dd9-844e-44b6-8b75-21a99a7d6911)